src/components/o-testimonials/_preview.html

```diff
@@ -0,0 +1,24 @@
+<axa-testimonials classes="o-testimonials"
+                  title="Customer Reviews"
+                  auto-rotate-enabled="true"
+                  auto-rotate-time="5000"
+                  subtitle="AXA works hard to provide the best service possible to its customers.">
+  <div class="o-testimonials__item js-o-testimonial__item">
+    1111 Very helpful once I got through to the correct person but I was constantly being transferred from person to person,
+    in the end I called into my local axa once in the city and got sorted in 10 minutes after 3 weeks of phone calls.
+    <br/>
+    ANDREW JACKSON, ADVOCATE
```

Comment:
this does not look as per design, can you check?

Reply:
I revised the designs, I will update the PR with the changes accordingly.